src/frontend/components/LiveVideoPanel/LiveVideoTabPanel/index.spec.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { render, screen } from '@testing-library/react';
+
+import { LivePanelItem } from 'data/stores/useLivePanelState';
+import { imageSnapshot } from 'utils/tests/imageSnapshot';
+import { wrapInIntlProvider } from 'utils/tests/intl';
+
+import { LiveVideoTabPanel } from '.';
+
+describe('<LiveVideoTabPanel /> titles', () => {
+  it('renders for item LivePanelItem.APPLICATION', () => {
+    render(
+      wrapInIntlProvider(
+        <LiveVideoTabPanel item={LivePanelItem.APPLICATION} selected={false} />,
+      ),
+    );
+
+    screen.getByText('application');
+  });
+
+  it('renders for item LivePanelItem.CHAT', () => {
+    render(
+      wrapInIntlProvider(
+        <LiveVideoTabPanel item={LivePanelItem.CHAT} selected={false} />,
+      ),
+    );
+
+    screen.getByText('chat');
+  });
+
+  it('renders for item LivePanelItem.JOIN_DISCUSSION', () => {
+    render(
+      wrapInIntlProvider(
+        <LiveVideoTabPanel
+          item={LivePanelItem.JOIN_DISCUSSION}
+          selected={false}
+        />,
+      ),
+    );
+
+    screen.getByText('viewers');
+  });
+});
+
+describe('<LiveVideoTabPanel /> styles', () => {
+  it('renders with default style when not selected, not hovered, not focused', async () => {
+    render(
+      wrapInIntlProvider(
+        <LiveVideoTabPanel item={LivePanelItem.APPLICATION} selected={false} />,
+      ),
+    );
+
+    const button = screen.getByRole('tab');
+
+    expect(button).toHaveStyle('background-color: rgb(255, 255, 255);');
+    expect(button).toHaveStyle('flex: 1;');
+    expect(button).toHaveStyle('box-shadow: inset 0 -1px #81ade6;');
+
+    const text = screen.getByRole('tab_title');
+
+    expect(text).toHaveStyle('color: rgb(129, 173, 230);');
+    expect(text).toHaveStyle('font-family: Roboto-Bold;');
+    expect(text).toHaveStyle('font-size: 10px;');
+    expect(text).toHaveStyle('font-weight: bold;');
+    expect(text).toHaveStyle('letter-spacing: -0.23px;');
+    expect(text).toHaveStyle('text-align: center;');
+    expect(text).toHaveStyle('text-transform: uppercase;');
+
+    await imageSnapshot();
+  });
+
+  it('renders with selected style', async () => {
+    render(
+      wrapInIntlProvider(
+        <LiveVideoTabPanel item={LivePanelItem.APPLICATION} selected={true} />,
+      ),
+    );
+
```

Comment:
Can you add to show the text is on with selected=true
 screen.getByText('viewers');

Reply:
https://github.com/openfun/marsha/pull/1248/commits/8f474e9ab19a885c606eb3e896d5498e83738a01

is that ok for you ? I haded test to check the right title is displayed whether selected is true or false